src/installation/guides/macbook.md

```diff
@@ -0,0 +1,53 @@
+# Macbookpro 12,1 (early 2015) 
```

Comment:
Is this specific to only one model?

Also, trailing whitespace here (if you end up with two spaces, it is significant in Markdown, so it should be avoided).

Reply:
i don't have many laptop, at least that's what i must do on mine to have it working.
it is also similar to what was described on the old wiki 'macbook' page. i believe he had a slightly different model than mine, but we got the same 'mouse problem'.
(pointer only moving up and down).

about the formating, here's my comment: it's a pain in the put, these modern tools.

Reply:
I've encountered the mouse issue on 2015 models, but what this document should maybe be is a series of problem/solution sections instead of targeting a specific model. Models can be called out if it's truly model-specific.